playbook/app/pb_kits/playbook/utilities/_hover.scss

```diff
@@ -2,11 +2,11 @@
 
 @mixin hover-color-classes($colors-list) {
     @each $name, $color in $colors-list {
-      .hover_background_#{"" + $name}:hover {
+      .hover_background-#{"" + $name}:hover {
```

Comment:
Is there a reason we changed the underscore for all those separators on these?

I'd prefer to stick to what we had, unless there's an issue you encountered with some scss bleed occurring. 

Preference:
```suggestion
      .hover_background_#{"" + $name}:hover {
```

It looks like you changed it to a `-` instead of underscore for background and color:

.hover_background-#{"" + $name}:hover {
.hover_color-#{"" + $name}:hover {

But, you left it unchanged for scale and shadow. 
If possible, we want color & background to match the way we have it for scale and shadow, with _all_  underscores for consistency. 

Thanks dude!

Reply:
@jasperfurniss 

The card kit has css to that "listens" for our color tokens and changes the background of the kit if they are present.

Using a different pattern, in this case `-` avoids that query 

this behavior in our card kit is the same behavior in our typography kits  which is why i use the same patter for hover

![Screenshot 2024-11-13 at 2 06 14 PM](https://github.com/user-attachments/assets/70cb2497-b6ff-4705-9cd9-8ca2b2fe83a0)